### ISSUE TITLE
Source of BS data?

### ISSUE THREAD

**Issue by munikarmanish** [amant/Nepali-Date-Convert#1]
Hi, can you provide the source of the BS data (list of number of days in every BS year/month) ?

**Comment by amant** [comment]
Hi Manish,
I can't seem to find the original excel file I got from one of the calendar publisher. Non the less it is in the code itself too as array.

line 9 through 100
- https://github.com/amant/Nepali-Date-Convert/blob/master/c%2B%2B/nepCalendar/nepali_calender.cpp

Hope that helps.
Cheers

**Comment by munikarmanish** [comment]
Hi @amant I already found the array in the source code. I just wanted to know why there's a limit (2000‒2090). :)

**Comment by amant** [comment]
Yes unfortunately the dataset I bought from one of the calendar publisher at that time only had upto 2090, which was good enough for my project then. May be you can contact them too.

**Comment by pawan-acharya** [comment]
![date](https://user-images.githubusercontent.com/69037187/90527530-5de4ba80-e191-11ea-8931-f6e51b38fe9a.PNG)
English to Nepali date are not correct and recurring, as u can see here

**Comment by amant** [comment]
You are right, that's incorrect. Something wrong in the date mappings. If you could create a PR, would really appreciate it :)
